guides/doc-Content_Management_Guide/topics/Importing_Red_Hat_Content.adoc

```diff
@@ -62,7 +62,7 @@ To change the default download policy for repositories, complete the following s
 . Change the default download policy depending on your requirements:
 +
 * To change the default download policy for Red Hat repositories, change the value of the *Default Red Hat Repository download policy* setting.
-* To change the default download policy for custom repositories, change the value of the *Default Custom Repository download policy* setting.
+* To change the default download policy for {customrepos}, change the value of the *Default Custom Repository download policy* setting.
```

Comment:
You used {customrepos} instad of {customrepo} incouple places. Need to be {customrepo} everywhere.

Reply:
Wonders never cease. Fixed.